cranelift-codegen/src/isa/mod.rs

```diff
@@ -22,7 +22,6 @@
 //! ```
 //! # extern crate cranelift_codegen;
 //! # #[macro_use] extern crate target_lexicon;
```

Comment:
Should we remove these two commented-out lines as well?

Reply:
Those are not commented out, they are just marked as invisible on rustdoc render.

Reply:
Cool, didn't know about that, 👍